### DISCUSSION TITLE
Hybrid search view

### DISCUSSION THREAD

**Discussion by mlazowik** [po8klasie/po8klasie#78] [Ideas]

**Comment by karoli-na** [comment]
Paweł wczoraj zainspirował mnie tymi referencjami z serwisów do wyszukiwania nieruchomości i zaczęłam grzebać trochę głębiej. A co jakby połączyć widok mapy z listą, tak jak jest np. na Airbnb? [https://www.airbnb.pl/s/Krak%C3%B3w/homes?tab_id=all_tab&refinement_paths%5B%5D=%2Fhomes&query=Krak%C3%B3w&place_id=ChIJ0RhONcBEFkcRv4pHdrW2a7Q&source=structured_search_input_header&search_type=search_query](url)
Co do działania tego to już musielibyśmy pewnie ustalić, czy np. przesuwanie mapy przeładowuje wyniki na liście. Ewentualnie możemy zrobić tak, że lista i mapa nie mają wpływu na siebie nawzajem, ale filtry działają na nie obie. Oczywiście, do przemyślenia w ogóle czy byśmy coś takiego chcieli wykorzystać i czy w ogóle jesteśmy w stanie to zrobić :) Załączam przykładowy widok do tego ekranu :)
![Lista szkol – 1](https://user-images.githubusercontent.com/66380837/83806868-6cebbe80-a6b2-11ea-9bb0-2f4be68806e7.png)

**Comment by micorix** [comment]
Obecnie zaimplentowaliśmy hybrydowy widok jako jedyny widok wyszukiwania. Locking
